apps/fz_http/priv/repo/migrations/20221126044850_create_api_tokens.exs

```diff
@@ -0,0 +1,16 @@
+defmodule FzHttp.Repo.Migrations.CreateApiTokens do
+  use Ecto.Migration
+
+  def change do
+    create table(:api_tokens, primary_key: false) do
+      add(:id, :uuid, primary_key: true)
+      add(:revoked_at, :utc_datetime_usec)
+      add(:user_id, references(:users, on_delete: :nothing), null: false)
```

Comment:
For my own understanding: Why wouldn't we also remove a user's tokens when the user is deleted?

Reply:
Well we are (see the schema file) -- just not using a cascading delete on the index. I think I was just following the precedent set in other migrations.

Reply:
Ahhh okay that makes sense!